test/plugin/test_compressable.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../helper'
+require 'fluent/plugin/compressable'
+
+class CompressableTest < Test::Unit::TestCase
+  include Fluent::Plugin::Compressable
+
+  sub_test_case '#compress' do
```

Comment:
Fixed data source and compressed one can be prepared as instance variables like `@src` & `@gzip` in `setup`.